src/ptvsd/common/launcher.py

```diff
@@ -0,0 +1,69 @@
+import sys
```

Comment:
Needs the copyright header and `from __future___`

(The second part isn't really necessary, but I'd rather keep it uniform so that we don't forget to add it when it does become necessary for new additions to the file.)

Reply:
Oh, and can you also run Black on it for the initial commit?